components/DocTable.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
```

Comment:
Can we refine the style of the table to be similar to this one  (@DhairyaMajmudar )

https://developer.android.com/about/versions/15/release-notes?hl=es-419